Wording fix in interact with charts doc (#15040)
netdata/netdata: docs/cloud/visualize/interact-new-charts.md

   At least one of the dimensions in the chart has partial data, meaning that not all instances available contributed data to this point. This can happen when a container is stopped, or when a node is restarted. This indicator helps to gain confidence of the dataset, in situations when unusual spikes or dives appear due to infrastructure maintenance, or due to failures to part of the infrastructure.

2. **[O]: Overflown**

>    At least one of the datasources included in the chart has a counter that indicates it has overflown at that exact point.

Reviewer: ```suggestion
   At least one of the data sources included in the chart has a counter that has overflowed at this point.
```

An attempt to simplify the sentence.
